components/search_results_item/search_results_item.jsx

```diff
@@ -8,9 +8,10 @@ import {Posts} from 'mattermost-redux/constants/index';
 import * as ReduxPostUtils from 'mattermost-redux/utils/post_utils';
 import {OverlayTrigger, Tooltip} from 'react-bootstrap';
 
+import CommentIcon from 'components/common/comment_icon.jsx';
```

Comment:
I don't know why you are moving it here, but is not wrong, but you need to remove the extension, because you are renaming the file to `.tsx`